lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtRouting.java

```diff
@@ -136,7 +136,12 @@ public PtRoutingResponse findPtRoute(PtRoutingRequest request) {
         );
         // ghRequest.setBlockedRouteTypes(request.getRoutingParameters().excludedPtModes);//FIXME generalize this
         ghRequest.setEarliestDepartureTime(departureTime);
-        ghRequest.setWalkSpeedKmH(SpeedUtils.ms2kmh(request.getRoutingParameters().getWalkingSpeedMps()));
+
+        if (request.getRoutingParameters().getWalkingSpeedMps() == null) {
```

Comment:
Wouldn't it be easier to initialize the walking speed in the `PtRoutingParameters` with a default value?

Reply:
For correctness, we need to use the speed of the mapping config as default. That's not possible by defining it in PtRoutingParameters.